tmt/steps/execute/internal.py

```diff
@@ -151,11 +153,31 @@ def execute(self, test: Test, guest: Guest,
         # Create data directory, prepare test environment
         environment = self._test_environment(test, extra_environment)
 
+        test_wrapper_filepath = os.path.join(workdir, TEST_WRAPPER_FILENAME)
+
         # Prepare the test command (use default options for shell tests)
         if test.framework == "shell":
-            command = f"{tmt.utils.SHELL_OPTIONS}; {test.test}"
+            test_command = f"{tmt.utils.SHELL_OPTIONS}; {test.test}"
+        else:
+            test_command = test.test
+
+        self.write(test_wrapper_filepath, test_command, 'w')
```

Comment:
Perhaps we should add a `self.debug()` line with the test command here? Because now the log only contains:

    Run command: set -eo pipefail; ./tmt-test-wrapper.sh </dev/null |& cat

Which is always the same. No information about what the actual test script was.

Reply:
Added in 45e9bd5c654d8c31645fc2e630f2d4ebcfdbd28d.